examples/Demo/Shared/App.razor.cs

```diff
@@ -0,0 +1,9 @@
+namespace FluentUI.Demo.Shared;
+public partial class App
+{
+    public const string MESSAGES_NOTIFICATION_CENTER = "NOTIFICATION_CENTER";
```

Comment:
We'll need to create another PR to standardize constant naming conventions (upper / pascal).